descope/sdk/mgmt.go

```diff
@@ -668,11 +668,14 @@ type Project interface {
 	// - This action is supported only with a pro license or above.
 	// - Users, tenants and access keys are not cloned.
 	// Returns The new project details (name, id, and tag).
-	Clone(ctx context.Context, name string, tag descope.ProjectTag) (*descope.CloneProjectResponse, error)
+	Clone(ctx context.Context, name string, tag descope.ProjectTag, customTags []string) (*descope.CloneProjectResponse, error)
```

Comment:
I don't think clone should receive custom tags since we said that it should be taken from the source project (unlike the `tag` )

Reply:
it was part of the requirments

Reply:
this means that cloning project from the UI will loose it tags, it doesn't make sense that you both get the custom tags from API and from the imported project, we need to decide either we get it as an argument or we take if from the source project (or we do some sufisticated merge, but I really i don't recommend to do that)